zebra-chain/src/block.rs

```diff
@@ -143,6 +145,18 @@ impl Block {
             .map(|transaction| transaction.orchard_nullifiers())
             .flatten()
     }
+
+    /// Get all the value balances from this block by summing all the value balances
+    /// in each transaction the block has.
```

Comment:
Missing documentation: which UTXOs need to be in `utxos`?

Please also make this change in PR #2486.